packages/sit-onyx/src/components/OnyxListbox/OnyxListbox.vue

```diff
@@ -162,6 +183,28 @@ const isEmpty = computed(() => props.options.length === 0);
     max-width: 20rem;
     font-family: var(--onyx-font-family);
 
+    &__wrapper {
+      max-height: calc(var(--max-options) * var(--option-height));
+      overflow: auto;
+      outline: none;
+    }
+
+    &__group {
+      padding: 0;
+
+      &:not(:last-child) {
+        border-bottom: 1px solid var(--onyx-color-base-neutral-300);
+        margin-bottom: var(--onyx-spacing-4xs);
```

Comment:
```suggestion
        margin-bottom: var(--onyx-spacing-2xs);
```

The separator is 8px high in Figma :)